polynote-frontend/polynote/cell.js

```diff
@@ -519,4 +567,9 @@ export class TextCell extends Cell {
     get content() {
         return this.editor.markdownContent;
     }
+
+    applyEdits(edits) {
+        // TODO
```

Comment:
What is this TODO for?

Reply:
Uhh... hmm. Not sure. I'll remove it.

Reply:
Oh, no. It's just saying that `applyEdits` needs to be implemented in `TextCell`. I'll clarify.